### ISSUE TITLE
XmlBuilder does not generate second ? in doc type

### ISSUE THREAD

**Issue by mspanc** [joshnuss/xml_builder#10]
```
iex(2)> {:a} |> XmlBuilder.doc
"<?xml version=\"1.0\">\n<a/>"
```

```
iex(3)> XmlBuilder.generate(:_doc_type, 0)
"<?xml version=\"1.0\">"
```

Second `?` is in the first line is missing.

Elixir 1.0.5, xml_builder 0.0.6

**Comment by mspanc** [comment]
I've noticed that it's fixed in https://github.com/joshnuss/xml_builder/commit/0d4b0a965d4962d1f7b7967211550d428bf76fbf but since it's rather major flaw and commit was made in July, you can make a new release? :)

**Comment by joshnuss** [comment]
Thanks for reporting that @mspanc,

Just published [0.0.7](https://hex.pm/packages/xml_builder/0.0.7)

**Comment by mspanc** [comment]
Thanks!
